CefSharp.Core.Runtime/Internals/CefBrowserHostWrapper.cpp

```diff
@@ -391,10 +391,19 @@ void CefBrowserHostWrapper::SendKeyEvent(int message, int wParam, int lParam)
             // 2 Either CTRL key is pressed.
```

Comment:
Ideally this comment is updated to include the new Shift behaviour.

Reply:
Done